app/src/main/java/com/yeonae/chamelezone/data/source/remote/place/PlaceRemoteDataSourceImpl.kt

```diff
@@ -12,25 +12,26 @@ import okhttp3.ResponseBody
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
+import java.math.BigDecimal
 
 class PlaceRemoteDataSourceImpl private constructor(private val placeApi: PlaceApi) :
     PlaceRemoteDataSource {
     override fun registerPlace(
-        keywordName: String,
+        keywordName: MutableList<Int>,
         name: String,
         address: String,
-        openingTime: String,
+        openingTime: MutableList<String>,
```

Comment:
```suggestion
        openingTime: List<String>,
```
수정될 가능성이 있는 List type으로 가져오면, 데이터의 안정성을 보장하기 어려울 것 같아요.

Reply:
수정했습니다